runtime/resolvers/legacy_metrics.go

```diff
@@ -327,20 +331,33 @@ func (r *legacyMetricsResolver) formatValue(f formatter.Formatter, v any) any {
 type legacyResolverResult struct {
 	data   []byte
 	schema *runtimev1.StructType
+
+	parsed []map[string]any
```

Comment:
Nit: `rows` to match with other resolver results ?

Reply:
Fixed